packages/react/.storybook/preview.jsx

```diff
@@ -271,6 +270,14 @@ export const decorators = [
       document.body.setAttribute('data-dark-theme', darkTheme)
     }, [colorScheme])
 
+    // Set data-a11y-link-underlines=true to enable underlines in all stories except the Link DevOnly Inline Story.
```

Comment:
The Link Devonly inline story has examples to show what impact this underline data attribute has. 

Adding this conditional to avoid ending up with this markup on that particular story, because it results in all links being underlined.=:

```.tsx
<div data-a11y-link-underlines="true">
  ... 
  ...
  <div data-a11y-link-underlines="false">
     Learn more <Link href="#">about GitHub</Link>
  </div>
```